packages/core/src/colors/index.ts

```diff
@@ -1,6 +1,11 @@
-import type { LikeC4Theme } from '../types/theme'
+import type {
+  HexColorLiteral,
+  LikeC4Theme,
+  ThemeColorValues,
+} from '../types/theme'
 import { ElementColors } from './element'
 import { RelationshipColors } from './relationships'
+import { generate } from '@ant-design/colors'
```

Comment:
I suggest to use [`@mantine/colors-generator`](https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/colors-generator/src/generate-colors.ts#L44C17-L44C31)

This is how it works: https://mantine.dev/colors-generator/

Why?
Initially I used tailwind, and picked colors manually, thats why I have these constants here. I am not good in knowing the aspects, and seems picking something like [Open Colors](https://yeun.github.io/open-color/), [Radix](https://www.radix-ui.com/colors) or [Adobe Spectrum](https://spectrum.adobe.com/page/color-system/)

At least for now I suggest to stay with mantine scale.